client/app/components/AppContainer.jsx

```diff
@@ -2,14 +2,44 @@ import React from 'react';
 import { Grid, Col } from 'react-bootstrap';
 import Calendar from './calendar/Calendar';
 import SideNav from './layout/SideNav';
+import CreateEventModal from './CreateEventModal';
 
-export default (props) => (
-  <Grid>
-    <Col xs={12} md={2}>
-      <SideNav />
-    </Col>
-    <Col xs={12} md={10}>
-      <Calendar {...props} />
-    </Col>
-  </Grid>
-);
+
+export default class AppContainer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      showModal: false
+    };
+
+    this.openModal = this.openModal.bind(this);
```

Comment:
You can use `_.bindAll` here to be consistent with https://github.com/u2i/confy/pull/51/files#diff-e7249ecfd1c2533b4881b03af541ec00R32.